src/bosh-director/lib/bosh/director/instance_updater.rb

```diff
@@ -104,8 +104,8 @@ def update(instance_plan, options = {})
 
         release_obsolete_ips(instance_plan)
 
-        update_dns(instance_plan)
         @disk_manager.update_persistent_disk(instance_plan)
+        update_dns(instance_plan)
```

Comment:
Possible to test a scenario where persistent disk updates fail, and then ensure the "latest" DNS version is the version prior to the update?